### DISCUSSION TITLE
Student developer pack is not activated

### DISCUSSION THREAD

**Discussion by Impavan007** [github/india#747] [Open Source]
Is there any one who can help me to activate my student developer pack I have applied for it several times also I have updated my LinkedIn my skills with Azure still not activated 
![Screenshot_2022-01-29-11-44-05-121_com android chrome](https://user-images.githubusercontent.com/96287711/151650053-59faf2b1-3e9e-4a97-ab88-77c841971452.jpg)
Also how to scan this QR code

**Comment by FRTMentor** [comment]
From FRT we are not supporting the activation of the Developer pack.
Please reach out the GitHub support team for more information.
